ios/mandelbrot/mandelbrot/Mandelbrot.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  Mandelbrot.swift
+//  mandelbrot
+//
+//  Created by Leon von Tippelskirch on 11.01.23.
+//
+
+import Foundation
+import UIKit
+
+struct Mandelbrot {
+    static func echo(str :String) -> String {
+        
+        let result = rust_echo(str)
+        
+        let swift_result = String(cString: result!)
+        
+        rust_echo_free(UnsafeMutablePointer<Int8>(mutating: result))
+        
+        return swift_result
+    }
+    
+    static func render() -> UIImage? {
+            
+        let width = 384
+        let height = 384
+        
+        let buffer = rust_render(Int32(width), Int32(height))
+    
+        let renderer = UIGraphicsImageRenderer(size: CGSize(width: width, height: height))
+
+        let img = renderer.image { ctx in
+            ctx.cgContext.setStrokeColor(UIColor.black.cgColor)
+            
+            for x in 0...width-1 {
+                for y in 0...height-1 {
+                    let currentBuffer = (buffer?[x*width+y] ?? 255)
+                    
+                    ctx.cgContext.setStrokeColor(CGColor(gray: CGFloat(currentBuffer)/255.0, alpha: 1.0))
```

Comment:
I had to convert the buffer on android first to take care of the intensity:
https://github.com/thebino/mobile-rust-mandelbrot/blob/main/android/app/src/main/kotlin/pro/stuermer/mandelbrot/ui/HomeViewModel.kt#L67